Restaurant Page project is finished

elmejdki opened the pull request:
I kindly request to review my project :smiley: this is the JavaScript Restaurant page project, where have followed the next steps:
:point_down: :point_down: :point_down: :point_down:
- I have used ES6 modules,
- Set up WebPack properly,
- Developed the Four pages using vanilla JavaScript,
- And deployed to GitHub pages as suggested by the OdinProject.

elmejdki committed:
Remove unecessary screenshot image

elmejdki committed:
Configure linters

elmejdki committed:
Configure webpack

elmejdki committed:
Add Home Page

elmejdki committed:
Fix Home page style

elmejdki committed:
Ignore experiments directory

elmejdki committed:
Change page title

elmejdki committed:
Add menu page

elmejdki committed:
Fix menu style

elmejdki committed:
Delete an unnecessary json file

elmejdki committed:
Add about page

elmejdki committed:
Update the about background image

elmejdki committed:
Add contact page

elmejdki committed:
Minimize picture to a better size

elmejdki committed:
Initial build subtree commit

elmejdki committed:
Delete the build folder to redo the deployment step

elmejdki committed:
Add a new font

elmejdki committed:
Update Readme file

elmejdki committed:
Update the live demo link in the readme file

sumancrest0001 commented:
**Status: Changes required :gear:
Hello @elmejdki,** Your file structure and code look good but I had a problem while running it. There are small changes required to get approved. Let's consider the following points.

[Required] The built js file should be main.js, not bundle.js. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png)

[Required] I got the following error while running the project. Let me know whether it is running correctly on your machine or not.
![Screenshot from 2020-10-03 09-33-04](https://user-images.githubusercontent.com/37747619/94982690-59673c00-055c-11eb-83dc-2a783f5fac76.png)

[Required] Please add installation and running steps in the readme file to run the project.

[Optional] Live link is not added correctly in the readme file. Please add if you have.

sumancrest0001 reviewed (changes requested):
**Make the required changes and send it for another review. If you have any query feel free to message me on slack. Suman Shrestha
Happy coding !!!**

elmejdki committed:
Change the JavaScriot generated name to main.js

elmejdki committed:
Added instaltion and deployments steps

elmejdki commented:
## Hi, @sumancrest0001 thank you very much for your code review.

- I have changed the bundle file name to main.js

- eventually, I don't have that error at all, maybe you forgot to run npm install that's why you are getting that error.

- I have added the installation and running steps to the readme file.

- for the live link I had a typo before now it's fixed, you can check it.

### Best Regards.

addod19 commented on README.md:
```diff
@@ -1,57 +1,71 @@
 
-### Usage
+Congrats the project is fully working.
 
-### Run tests
+_You may have some issues with the `image-webpack-loader`  but that depend on your operating system each operating system has his own way to solve if you goggled it just for some couple minutes I'm sure you will find the solution, if not you can contact me any time you need I will be happy to help, my contact info are below_
```
[OPTIONAL]
- I think this part of the text should rather be in the PR. Congratulations :fireworks:

elmejdki replied:
_Hi, I have already thought of that, to be honest, but not every user of the app will look in the PR's most of the users read the readme only._

addod19 reviewed (approved):
## Status: Approved :trophy: :balloon: 

Hi @elmejdki, congratulations :100: Having fixed all the comments made by the previous TSE :+1: I HEREBY APPROVE THE PROJECT. Congratulations go ahead and merge into development :1st_place_medal: 

_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)
